packages/core/src/extensions/SuggestionMenu/getDefaultEmojiPickerItems.ts

```diff
@@ -0,0 +1,37 @@
+import data, { Emoji, EmojiMartData } from "@emoji-mart/data";
+import { init, SearchIndex } from "emoji-mart";
+
+import { BlockSchema, InlineContentSchema, StyleSchema } from "../../schema";
+import { BlockNoteEditor } from "../../editor/BlockNoteEditor";
+import { checkDefaultInlineContentTypeInSchema } from "../../blocks/defaultBlockTypeGuards";
+import { DefaultGridSuggestionItem } from "./DefaultGridSuggestionItem";
+
+const emojiMartData = data as EmojiMartData;
```

Comment:
TBD: this might be a significant increase in package size

Reply:
package size grows from 330kb to 440kb. Not ideal, but also not a dealbreaker at this moment.

It could be improved by adding support for dynamic data loading (https://github.com/missive/emoji-mart?tab=readme-ov-file#-data)